app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,20 @@
+class TagsController < ApplicationController
+  include PageableConcern
+  before_action :set_tag, only: :show
+
+  # GET /tags
+  def index
+    @tags = paginate Tag.order_by_most_posts, 10000
```

Comment:
I also removed the "order_by_most" method, for ordering use the order service, I added it yesterday.